src/pages/permissions/CreateIdentityModal.tsx

```diff
@@ -60,11 +26,15 @@ const CreateIdentityModal: FC<Props> = ({ onClose }) => {
     }
   };
 
+  const handleClose = () => {
+    onClose();
+  };
```

Comment:
```suggestion
```
Can we directly call `onClose` instead? I don't think we need the extra callback here.